src/DssSpell.t.sol

```diff
@@ -607,97 +573,99 @@ contract DssSpellTest is DssSpellTestBase {
-            Payee(wallets.addr("WBC"),              9.92 ether),   // note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("LIBERTAS"),         9.92 ether),   // note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("CODEKNIGHT"),       9.92 ether),   // note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("PBG"),              9.92 ether),   // note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("FLIPFLOPFLAP_2"),   9.92 ether),   // note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("BANDHAR"),          9.92 ether)    // note: ether is a keyword helper, only MKR is transferred here
+        Payee[16] memory payees = [
+            Payee(wallets.addr("DECO_WALLET"),     125    ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("DUX_WALLET"),       56.48 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("DEFENSOR"),         29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("BONAPUBLICA"),      29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("QGOV"),             29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("TRUENAME"),         29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("UPMAKER"),          29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("VIGILANT"),         29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("WBC"),              20.16 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("PBG"),               9.92 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("BANDHAR"),           7.68 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("LIBERTAS"),          7.04 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("PALC"),              2.24 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("HARMONY"),           1.92 ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("VOTEWIZARD"),        1.6  ether), // note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("NAVIGATOR"),         0.32 ether)  // note: ether is a keyword helper, only MKR is transferred here
```

Comment:
Visual alignment is weird here:

```suggestion
         Payee[16] memory payees = [
            Payee(wallets.addr("DECO_WALLET"), 125 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("DUX_WALLET"),  56.48 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("DEFENSOR"),    29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("BONAPUBLICA"), 29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("QGOV"),        29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("TRUENAME"),    29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("UPMAKER"),     29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("VIGILANT"),    29.76 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("WBC"),         20.16 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("PBG"),         9.92 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("BANDHAR"),     7.68 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("LIBERTAS"),    7.04 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("PALC"),        2.24 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("HARMONY"),     1.92 ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("VOTEWIZARD"),  1.6  ether), // note: ether is a keyword helper, only MKR is transferred here
            Payee(wallets.addr("NAVIGATOR"),   0.32 ether)  // note: ether is a keyword helper, only MKR is transferred here
        ];
```

Reply:
I see that you are visually aligning the numbers by their precision.
IMO this is good when you have to compare numbers between themselves, but this is not the case. The numbers in each line have no relation to each other... we just need to make sure the address in the first argument is getting the right value as the second argument.

Still, I'll leave this decision to you.